src/components/structures/MatrixChat.js

```diff
@@ -584,6 +588,44 @@ module.exports = React.createClass({
     _onLoadCompleted: function() {
         this.props.onLoadCompleted();
         this.setState({loading: false});
+
+        // set up the right theme.
+        // XXX: this will temporarily flicker the wrong CSS.
+        dis.dispatch({
```

Comment:
oh, to make it the same codepath as setting the option in the settings dialog I suppose.